pkg/scheduler/scheduler_test.go

```diff
@@ -3651,15 +3651,16 @@ func TestRequeueAndUpdate(t *testing.T) {
 	w1 := utiltesting.MakeWorkload("w1", "ns1").Queue(q1.Name).Obj()
 
 	cases := []struct {
-		name              string
-		e                 entry
-		wantWorkloads     map[string][]string
-		wantInadmissible  map[string][]string
-		wantStatus        kueue.WorkloadStatus
-		wantStatusUpdates int
+		name                        string
+		e                           entry
+		wantWorkloads               map[string][]string
+		wantInadmissible            map[string][]string
+		wantStatus                  kueue.WorkloadStatus
+		wantStatusUpdates           int
+		wantResourceRequestsSummary bool
```

Comment:
Naming it as `resourceRequestsSummary` would be better since we typically add a `want` or `expected` prefix only to the desired one, but this is not the desired one, and it is input.

```suggestion
		resourceRequestsSummary bool
```